ut/client_ut.cpp

```diff
@@ -1210,3 +1210,87 @@ INSTANTIATE_TEST_SUITE_P(ClientLocalFailed, ConnectionFailedClientTest,
         ExpectingException{"Authentication failed: password is incorrect"}
     }
 ));
+
+
+class ConnectionSuccessTestCase : public testing::TestWithParam<ClientOptions> {};
+
+TEST_P(ConnectionSuccessTestCase, SuccessConnectionEstablished) {
+    const auto & client_options = GetParam();
+    std::unique_ptr<Client> client;
+    
+    try {
+        client = std::make_unique<Client>(client_options);
+        SUCCEED();
+    } catch (const std::exception & e) {
+        FAIL() << "Got an unexpected exception : " << e.what();
+    }
+}
+
+
+INSTANTIATE_TEST_SUITE_P(ClientMultipleEndpoints, ConnectionSuccessTestCase,
+    ::testing::Values(ClientCase::ParamType{
+        ClientOptions()
+            .SetHosts({
+                    getEnvOrDefault("CLICKHOUSE_HOST",     "somedeadhost"),
```

Comment:
Please use  plain `"somedeadhost"` value instead (without `getEnvOrDefault`), so it remains invalid host name on CI/CD.